engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -440,11 +214,48 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   onTransition {
-    case _ -> WorkflowExecutionSuccessfulState =>
-      pushWorkflowOutputMetadata(nextStateData)
-      context.parent ! WorkflowExecutionSucceededResponse(nextStateData.executionStore, nextStateData.outputStore)
     case _ -> WorkflowExecutionAbortedState =>
-      context.parent ! WorkflowExecutionAbortedResponse(nextStateData.executionStore, nextStateData.outputStore)
+      context.parent ! WorkflowExecutionAbortedResponse(nextStateData.jobExecutionMap)
+  }
+
+  private def handleNonRetryableFailure(stateData: WorkflowExecutionActorData, failedJobKey: JobKey, reason: Throwable, jobExecutionMap: JobExecutionMap) = {
+    val mergedStateData = stateData.mergeExecutionDiff(WorkflowExecutionDiff(Map(failedJobKey -> ExecutionStatus.Failed)))
+        .removeCallExecutionActor(failedJobKey)
+        .addExecutions(jobExecutionMap)
+
+    if (workflowDescriptor.getWorkflowOption(WorkflowFailureMode).contains(ContinueWhilePossible.toString)) {
+      mergedStateData.workflowCompletionStatus match {
+        case Some(completionStatus) if completionStatus == Failed =>
+          context.parent ! WorkflowExecutionFailedResponse(stateData.jobExecutionMap, reason)
+          goto(WorkflowExecutionFailedState) using mergedStateData
+        case _ =>
+          stay() using startRunnableScopes(mergedStateData)
+      }
+    } else {
+      context.parent ! WorkflowExecutionFailedResponse(stateData.jobExecutionMap, reason)
+      goto(WorkflowExecutionFailedState) using mergedStateData
+    }
+  }
+  
+  private def handleWorkflowSuccessful(data: WorkflowExecutionActorData) = {
+    import cromwell.util.JsonFormatting.WdlValueJsonFormatter._
+    import spray.json._
+    
+     val (response, finalState) = workflowDescriptor.workflow.evaluateOutputs(
+      workflowDescriptor.inputs,
+      data.expressionLanguageFunctions,
+      data.outputStore.fetchCallOutputEntries
+    ) map { workflowOutputs =>
+       workflowLogger.info(s"Workflow ${workflowDescriptor.workflow.unqualifiedName} complete. Final Outputs: \n" + workflowOutputs.toJson.prettyPrint)
```

Comment:
Use interpolation instead of concatenation.

```scala
workflowLogger.info(
  s"""Workflow ${workflowDescriptor.workflow.unqualifiedName} complete. Final Outputs:
      |${workflowOutputs.toJson.prettyPrint}""".stripMargin)
```